src/components/views/right_panel/UserInfo.tsx

```diff
@@ -499,16 +503,20 @@ interface IPowerLevelsContent {
     redact?: number;
 }
 
-const isMuted = (member: RoomMember, powerLevelContent: IPowerLevelsContent) => {
+export const isMuted = (member: RoomMember, powerLevelContent: IPowerLevelsContent) => {
     if (!powerLevelContent || !member) return false;
 
     const levelToSend =
         (powerLevelContent.events ? powerLevelContent.events["m.room.message"] : null) ||
         powerLevelContent.events_default;
+
+    if (!levelToSend) return false;
```

Comment:
I thought this was clearer than a `Number(levelToSend)` in the return.

`levelToSend` can be a number or undefined, and trying to coerce undefined to be a number gives `NaN`, which I think would always make the return check evaluate to false, regardless of `member.powerLevel`.

Assuming my understanding above is correct, this line preserves the behaviour of this function. Whether or not it should always return false if `levelToSend` is undefined is, I think, a separate question.

Reply:
You're probably right that you're preserving behaviour, but:

It's possible for powerlevels to be negative, so in theory `levelToSend` could be zero and `member.powerLevel` could be negative, in which case this function should return true.

I think what we really want here is `?? 0` on the end of the `levelToSend` calculation ? (For that matter, the `||` in the calculation should be `??` too.)

Maybe fixing this is a separate PR though...

Reply:
I did mean to ask if power level could be less than zero.

I've done what I think is the most sensible here which is change the check to be an explicit check of whether it is undefined or not, adding a comment to explain why the check is there. Happy to change it again if required.